extensions/sync-data-transfer/sync-data-transfer-api/src/main/java/org/eclipse/dataspaceconnector/transfer/sync/api/controller/SyncDataTransferValidationApiController.java

```diff
@@ -35,7 +35,7 @@ public SyncDataTransferValidationApiController(Monitor monitor, TokenValidationS
     @GET
     public Response validate(@HeaderParam("Authorization") String token) {
         monitor.info("Received request for token validation");
-        var result = tokenValidationService.validate(token);
+        var result = tokenValidationService.validate(token, null);
```

Comment:
Could have sense to have a default implementation for the `validate` method that calls the other passing an empty map? In every case I would avoid passing `null`

Reply:
Done (please check again if ok)